### PR TITLE
release slack upsell widget from experiment

### PR DESCRIPTION
**Pull request by camhashemi**
@cvializ some minor formatting tweaks, the SVG's width was being altered depending on the number of lines in `kf-fu-header` so I split it out.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/home/featureUpsell.tpl.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <div class="kf-feature-upsell" ng-show="showFeatureUpsell">
   <div class="kf-fu-slack">
     <span class="kf-fu-hide" ng-click="hide()">X</span>
-    <span class="kf-fu-header kf-flex-row">
-      <span class="svg-slack kf-fu-slack-icon"></span><b>Integrate <span ng-bind="orgToSync.name"></span> with Slack</b>
+    <span class="svg-slack kf-fu-slack-icon"></span>
+    <span class="kf-fu-header">
+      </span><b>Integrate <span ng-bind="orgToSync.name"></span> with Slack</b>
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/home/featureUpsell.tpl.html]: Does this intentionally close the span here?

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/home/featureUpsell.tpl.html]: Agreed looks like a bug.

Also consider removing the `<b>` tag and add `font-weight: bold` to the `.kf-fu-header` class's rules.

**Comment by cvializ** [comment]
Small thoughts, but besides that looks good :+1: :tada:

**Comment by camhashemi** [comment]
@cvializ updated

**Pull request merged**